draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1565,11 +1592,13 @@ Each variant can be used as a MAC by calling the `Encrypt()` function with the m
 
 As shown in {{D23}}, AEGIS-128X and AEGIS-256X share the same security properties and requirements as AEGIS-128L and AEGIS-256 respectively. In particular, the security level and usage limits remain the same.
 
+Without the ability to set the associated data, a successful forgery doesn't increase the probability of subsequent forgeries. Reforgeability reilience of AEGIS is analyzed in {{FLLW17}}.
```

Comment:
`doesn't` to `does not` and `The reforgeability reilience...`. It would be good if these two sentences could flow together a bit more somehow.